sources/Application/Views/BaseClasses/ModalView.cpp

```diff
@@ -59,18 +61,6 @@ void ModalView::SetWindow(int width, int height) {
   }
   ClearTextRect(-1, -1, width + 2, height + 2);
 
-  SetColor(CD_ACCENT);
-  GUITextProperties props;
-  props.invert_ = true;
-  char line[SCREEN_WIDTH + 1];
-  memset(line, ' ', SCREEN_WIDTH);
-  line[SCREEN_WIDTH] = '\0';
-  line[width + 4] = 0;
-  DrawString(-2, -2, line, props);
-  DrawString(-2, height + 1, line, props);
-  line[1] = 0;
-  for (int i = 0; i < height + 2; i++) {
-    DrawString(-2, i - 1, line, props);
-    DrawString(width + 1, i - 1, line, props);
-  }
+  SetColor(CD_NORMAL);
+  DrawUtils_Border(-2, -2, width + 3, height + 3, *this);
```

Comment:
Can we please just keep this PR to using the extended chars only for the vumeter, progress spinner and battery gauge.